notebook/Dockerfile

```diff
@@ -1,3 +1,4 @@
+ARG PYTHON_VERSION=3.8
```

Comment:
Should this be `ENV`?

Reply:
Gentle ping to change this to `ENV` :-)

Reply:
Thanks for the ping @seanfarley

Hrm I think `ARG` is the only instruction that can precede a `FROM`. If I update this to be `ENV PYTHON_VERSION=3.8`, then running `docker-compose build notebook` locally gives me:

```
Building notebook
ERROR: No build stage in current context
```

Reply:
I asked about this over in the Jupyter community discourse https://discourse.jupyter.org/t/specify-python-version-when-base-notebook-is-used-as-a-base-image/3732